tests/test_eventlog.py

```diff
@@ -40,12 +38,19 @@ def test_simple_databricks_log():
 
 def test_raw_databricks_log():
     event_log_path = Path("tests", "logs", "databricks.json").resolve()
+    event = get_event(event_log_path)
 
-    with tempfile.TemporaryDirectory() as temp_dir:
-        event_log = eventlog.EventLogBuilder(event_log_path.as_uri(), temp_dir).build()
+    assert all(
+        key in event
+        for key in ["Event", "Spark Version", "Timestamp", "Rollover Number", "SparkContext Id"]
+    ), "All keys are present"
```

Comment:
To clarify, the message states "All keys are present" on an assertion error. Shouldn't it be something like "A key was missing"?

Also, is it important to check for `SparkContext Id`?

Reply:
No need to check for SparkContext Id yet, might be something in the future though. Good catch on the error message.

Reply:
Yeah, good call out Neal. In the Java shop I worked at we used the message to describe the assertion like,
```
assertTrue(5 > 4, "5 is greater the 4");
assertNull("car is null", car);
```
(looks like the semantics were left to the organization though: https://softwareengineering.stackexchange.com/questions/301652/purpose-of-assertion-messages)

I don't remember the output when they failed, but in Python it looks like something that describes the error is better, e.g.
```
>>> assert "No rollover properties found in log file" == "Rollover file appears to be missing", "Exception message doesn't match"
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError: Exception message doesn't match
```

And the documentation appears to support that: https://docs.python.org/3/reference/simple_stmts.html#grammar-token-python-grammar-assert_stmt